### DISCUSSION TITLE
Shimmer - raise 15M proposal - SBot #230

### DISCUSSION THREAD

**Discussion by cometax** [iota-community/Shimmer-Governance#20] [Ideas]
What I aim with this proposal: 
1-	Provide the expected DAO funding while maintaining the initial thought of NFTs which I believe would bring a lot of outsiders to look into and participate in IOTA
2-	Generate initial increased demand for SMRs (3 years) by NFT holders – incentivized them to buy and hold SMR to maximize their boost effect
3-	Ensure the NFT holds value over time and does not go to 0 and at the same time, generates constant revenue to the DAO.  (Year 4 and after will have constant value)
4-	Provide similar ROI for each holder as each holder’s investment is the same across the board (would not be the case for the initial 3 years)
5-     Have a defined boost. 

Proposal:
Same number of NFTs as originally proposed (11,111) – stackable up to three per address - half the SMRs generated by the NFTs would go to the DAO.

Gold $3000 – 1000 NFTs  (plus 170 Gold air dropped to random account) – 6% boost for 3 years. Thereafter, 0% + 3000 SMRs per year increasing 8% every year to keep with inflation

Silver $2000 – 3000 NFTs   (plus 386 Silver air dropped to random account) – 4% boos for 3 years. Thereafter, 0% + 2000 SMRs per year increasing 8% every year to keep with inflation

Bronze $1000 – 6000 NFTs (plus 555 Bronze air dropped to random account) – 2% boost for 3 years. Thereafter, 0% + 1000 SMRs per year increasing 8% every year to keep with inflation

Note: Year 4 and on – NFTs will generated an estimated 1% of additional supply (20,000,000 SMRs year 4, 21,600,00 year 5 and so on…)
-	If interested, I ran multiple scenarios to simulate possible outcomes and can share the results and assumptions. 

Percentages above can be changed but the main idea is there. - Gold 6-12% and so on. The number of SMRs above was based on the assumption that 20,000,000 SMRs will equate to 1% of the supply at end of year 3, but this can be adjusted anywhere from 1-4% IMO to not have a major negative impact to the overall community.

**Comment by cometax** [comment]
I corrected a mistake I made just FYI. should be 6, 4, 2 % respectively

**Reply by Slum12** [reply]
I like the idea. But i would increase the flat amount by alot. If i get 3k SMR per year, you would get nearly the same by just buying Iota.
I would make it around 15k for gold, but also not increase it by year, because the SMR you get out of the NFT can also be used for staking.

But i like this proposal alot more then the IF proposal.

**Reply by cometax** [reply]
Yes, you can stake what you get out of the NFT; however if you don't adjust what it generates, the value of the nft won't be constant and would over time go to zero. In regards to your for first point, I want to clarify that it is 3000 MSMRs (same concept as MIotas). I'm sure that that was understood but wanted to clarify. If you assumed the value of SMR market to be for example 10% of the value of IOTA that means today, 1MSMR would be valued at $.15 so you would potentially get $450 a year in return (which means $225 for you and $225 for the DAO). It would take over 8 years to recover your investment which I think is a little below normal market returns. Not sure if this makes sense how I have explained it. I think we would loose a lot, if not all, of interest if the price was $15k, but I can certainly be wrong.

**Comment by cometax** [comment]
One item I would modify is how the NFT sale uses the term boost and instead convert it to an actual APY for easy of read. What this means for example with gold if the boost were to be 6%:
A regular holder gets 8% APY
A 1Gold NFT holders gets 8% + 0.48% APY = 8.48% (this is not the same as 8%+6%=14% APY)
A 3 Gold NFT holder gets 8% + 1.44% APY = 9.44% 

Note: I believe the DAO should vote on pre-defined options - 6%, 10%, or  15% boost. 6% boost (or .48% APY boost) should be the minimum.
